WordPress/Classes/Extensions/UIViewController+Additions.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+extension UIViewController {
+
+    /// Sets up a gesture recognizer to make tap gesture close the keyboard
+    func setupEditingEndingTapGestureRecognizer() {
```

Comment:
I'm not sure we want an extension for this. `RegisterDomainDetailsViewController` inherits `NUXTableViewController`, which inherits `NUXViewControllerBase`, which has these two methods: `setupBackgroundTapGestureRecognizer` and `handleBackgroundTapGesture`. It would be preferable to use that inherited functionality.

Reply:
I tried but setupBackgroundTapGestureRecognizer() method is not public in WordPressAuthenticatior project so I couldn't call it, let's take this on slack and see what is the best way to move forward about it